bigquery_etl/glam/templates/extract_probe_counts_v1.sql

```diff
@@ -17,6 +17,8 @@ WITH final_probe_extract AS ( SELECT
     MAX(total_users) as total_users,
     MAX(IF(agg_type = "histogram", mozfun.glam.histogram_cast_json(aggregates), NULL)) as histogram,
     MAX(IF(agg_type = "percentiles", mozfun.glam.histogram_cast_json(aggregates), NULL)) as percentiles,
+    MAX(IF(agg_type = "histogram", mozfun.glam.histogram_cast_json(non_norm_aggregates), NULL)) as non_norm_histogram,
```

Comment:
This is the last step of the ETL. The new columns `non_norm_histogram` and `non_norm_percentiles` are added the way `histogram` and `percentiles` are.